lib/url_helpers.rb

```diff
@@ -11,7 +11,7 @@ def url_active?(url, timeout_in_seconds=60)
       begin
         url = Addressable::URI.parse(url)
         response_code = Net::HTTP.start(url.host, url.port) {|http| http.head(url.path.blank? ? '/' : url.path).code}
-        active_status = %w(200 301 302)
+        active_status = %w[200 301 302 307 308]
```

Comment:
It looks like this is actually the wrong place for this change: I pulled the code and I still got an error message when trying to bookmark something on Mangadex. (It's possible this code is unused and can be removed.) However, bookmarking after updating app/validators/url_active_validator.rb instead works just fine.

It would be a good idea to add automated tests to spec/models/external_work_spec.rb to prove that the code works.

Reply:
> It's possible this code is unused and can be removed.

For the record, the `url_active?` method *is* called in `ExternalWork#set_url_status`, but `set_url_status` isn't called anywhere, so both methods should probably be deleted sooner or later.

Reply:
Oh that makes sense. Would you like me to remove the unused code in this PR or should I leave it and it can be taken out in a separate cleanup PR?

Reply:
Let's remove them in this PR.